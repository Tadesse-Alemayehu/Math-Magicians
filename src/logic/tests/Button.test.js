import renderer from 'react-test-renderer';
import Button from '../button';

it('render button', () => {
  const button = renderer
    .create(<Button />)
    .toJSON();
  expect(button).toMatchSnapshot();
});
