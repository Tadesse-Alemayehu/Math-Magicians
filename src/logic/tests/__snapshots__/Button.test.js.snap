// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render button 1`] = `
<button
  type="button"
/>
`;
